### PR TITLE
how to move cursor position within text field

### PR DESCRIPTION
**Pull request by roland9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,3 +140,14 @@ let start = firstCell.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
...
 ````
+
+### Move cursor position within text field
+
+In this example, a table view cell contains a text field with some content. To move the cursor within the text field, create a `XCUICoordinate` from the left edge of the text field, tap & hold and then drag `dx` points to the right.
+
+````swift
+let textField = cells.textFields.elementBoundByIndex(1)
+let start = textField.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
+let finish = start.coordinateWithOffset(CGVectorMake(70, 0))
```
Comment (joemasilotti) [inline README.md:151]: Does 70 _always_ work? I'm worried that a lowercase <kbd>l</kbd> might take up less space than an uppercase <kbd>W</kbd> and start to cause problems. Also, what about international characters?

**Comment by joemasilotti** [comment]
Thanks for this! I added a comment regarding the hard-coded value.

File: README.md (hunk 1 above)
Comment (roland9) [inline README.md:151]: you're right: the hard-coded 70 is only for illustration purpose - which might or might not work, depending on the text
I could add this to the paragraph, describing the context of the example - would that be clearer?
or, I could add a line of code to set the content of the textField - but I feel that would be distracting

Comment (joemasilotti) [inline README.md:151]: I prefer that the code be made generic enough to handle characters/fonts with widths no exactly equal to 70.

**Comment by joemasilotti** [comment]
I'm going to close this. Feel free to reopen if you can address my latest comment.

> I prefer that the code be made generic enough to handle characters/fonts with widths no exactly equal to 70.

**Pull request closed without merging**